examples/dataservice-write/example.cpp

```diff
@@ -68,14 +68,14 @@ int runExample() {
 
   // Check the response
   if (!response.IsSuccessful()) {
-    LOG_INFO_F("write-example",
```

Comment:
move "write-example" as `constexpr auto` to anonymous namespace

Reply:
Done